### PR TITLE
 Arg to create references 

### PR DESCRIPTION
**Pull request by mbounabi**
https://jira.kisio.org/browse/NAVP-1102

### CODE REVIEW

File: artemis/base_pytest.py [hunk 1]
```diff
@@ -258,99 +265,116 @@ def journey(self, _from, to, datetime,
...
-    # Get only the full_response part from the ref
-    ref_full_response = dict_ref['full_response']
+        def ref_resp2files(output_file, output_json):
+            """
+            Create a file for the filtered response and for the filtered response
+            """
+            with open(output_file, 'w') as reference_text:
+                reference_text.write(output_json)
+
+        def print_diff():
```
Comment (xlqian) [inline artemis/base_pytest.py]: can you put full_file_name_ref and full_file_name_resp into args plz?

File: artemis/base_pytest.py [hunk 2]
```diff
@@ -258,99 +265,116 @@ def journey(self, _from, to, datetime,
...
+            # open reference
+            with open(full_file_name_ref) as reference_text:
+                reference = reference_text.readlines()
+            # open response
+            with open(full_file_name_resp) as response_text:
+                response = response_text.readlines()
+
+            # Print failed test name
+            print_color('\n\n' + str(file_name) + ' failed :' + '\n\n', Colors.PINK)
+
```
Comment (xlqian) [inline artemis/base_pytest.py]:
```python
symbol2color = {'+': Colors.GREEN, '-': Colors.RED}
for line in difflib.unified_diff(reference, response):
    print_color(line, symbol2color.get(line[0], Colors.DEFAULT))
```

File: artemis/base_pytest.py [hunk 3]
```diff
@@ -258,99 +265,116 @@ def journey(self, _from, to, datetime,
...
+        Compare the response (which is a dictionary) to the reference
+        First, the function retrieves the reference then filters both ref and resp
+        Finally, it compares them
+        """
 
-    # Get only the full_response part from the ref
-    ref_full_response = dict_ref['full_response']
+        def ref_resp2files(output_file, output_json):
+            """
+            Create a file for the filtered response and for the filtered response
```
Comment (nonifier) [inline artemis/base_pytest.py]:
```suggestion
            Create a file for the filtered and the full response
```
Maybe ?

File: artemis/base_pytest.py (hunk 2 above)
Comment (nonifier) [reply artemis/base_pytest.py]: :rainbow:

File: artemis/base_pytest.py (hunk 3 above)
Comment (mbounabi) [reply artemis/base_pytest.py]: Actually: For the filtered **response** and filtered **reference**

File: artemis/base_pytest.py (hunk 1 above)
Comment (mbounabi) [reply artemis/base_pytest.py]: Sure :heavy_check_mark:

**Pull request merged**